### ISSUE TITLE
Check the project with a linter

### ISSUE THREAD

**Issue by znhv** [CiviWiki/OpenCiviWiki#1096]
### Idea summary

Fix all warnings in the project with the wemake-python-styleguide linter.

### Further details

I noticed that the project has a lot of inconsistencies with the generally accepted rules of code design. Can I use the **wemake-python-styleguide** style guide to fix problems and warnings in the project?

**Comment by brylie** [comment]
For our purposes, the [`black` code formatter](https://github.com/psf/black) should suffice. Also, we don't want to format auto-generated code, such as Django migrations.

**Comment by AbhijithGanesh** [comment]
Brylie I think we need to choose either black or flake, In the repo there is a .flake8 file for the same, if we are choosing black, we need to remove flake related files and need to specify the same on contributors guide @brylie and @znhv

**Comment by znhv** [comment]
That's right, I also think we need to standardize the code style. I am more inclined to flake8, it is more flexible and it is used in many projects.

**Comment by brylie** [comment]
I believe `flake8` and `black` can coexist. Since we are already using `black`, I'd like to stick with it. 

In order to harmonize `flake8` and `black`, we can add a `.flake8` configuration with the following:

```yaml
[flake8]
max-line-length = 88
extend-ignore = E203
```

## Resources

- [using black with flake8](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8)
- [Using black and flake8 in tandem](https://sbarnea.com/lint/black/)

**Comment by gorkemarslan** [comment]
@brylie, we have already configurations for `flake8`:
https://github.com/CiviWiki/OpenCiviWiki/blob/d9475ff12eb03a7748af82a7e1f244cefcc2d15b/project/setup.cfg#L1-L4

I tend to collect all config files in the `setup.cfg` file instead of using separate files for each configuration because they usually don't pass 4 or 5 lines. Getting together with [setup.cfg](https://flake8.pycqa.org/en/latest/user/configuration.html#project-configuration) results in a project that is in a good shape and well-structured, in my opinion. Otherwise, in the future, the bulk of files seems scary and is hard to follow. Actually, I don't want this project to have too many files like [EvalAI](https://github.com/Cloud-CV/EvalAI). Let's keep simplicity. 😄 

It might be worth reviewing issue #55.

**Comment by znhv** [comment]
And you can also disable unnecessary directories or files like this.
```
per-file-ignores =
  project/accounts/*.py: WPS421, Q000
```

**Comment by gorkemarslan** [comment]
`exclude` is used for this purpose whereas `per-file-ignores` is used to ignore specific warnings for desired files. I guess you mean the second one. I find `exclude` more comprehensive. For this project, I guess we don't need `per-file-ignores`.

**Comment by brylie** [comment]
> Getting together with setup.cfg results in a project that is in a good shape and well-structured, in my opinion. 

That sounds like a great improvement. I second the motion.
